src/main/pages/user-management/organizations.adoc

```diff
@@ -64,77 +81,119 @@ CHE_MAIL_SMTP_AUTH_USERNAME=no-reply@gmail.com
 CHE_MAIL_SMTP_AUTH_PASSWORD=password
 ----
 
-[id="create-sub-organization"]
-== Create Sub-Organization
+[id="creating-sub-organizations"]
+== Creating sub-organizations
+
+To create a sub-organization:
+
+*  Select the *Sub-Organizations* tab.
 
-The creation of sub-organization can be done from organization details page by selecting Sub-Organizations tab and clicking "Add Sub-Organization" button.
+*  Click the *Add Sub-Organization* button.
 
-The flow of sub-organization creation is the same as for organization.
+The steps to create a sub-organization are the same for creating an organization. Use them to create the organization.
```

Comment:
If the steps are the same, can we link to those steps? There's no guarantee that the reader is reading this document linearly. :)

Reply:
Not sure - I will look for the steps